### ISSUE TITLE
When Directory contains 2 csproject files (a new one and a old one), NuGet thinks it's project.json

### ISSUE THREAD

**Issue by jogibear9988** [Microsoft/msbuild#1924]
I've started to port my Software to NetStandart.

For this I've created a new csproj in the same directory. The new netstandart Project builds without problems in VS2017, but the 3 old Project files (wich are not modified), throw the following error:

(ResolveNuGetPackageAssets target) -> 
  C:\Program Files (x86)\Microsoft Visual Studio\2017\Community\MSBuild\Microsoft\NuGet\15.0\Microsoft.NuGet.targets(197,5): error : Your project is not referencing the ".NETFramework,Version=v4.0" framework. Add a reference to ".NETFramework,Version=v4.0" in the "frameworks" section of your project.json, and then re-run NuGet restore. [C:\projects\migrator-net\src\Migrator.Framework\DotNetProjects.Migrator.Framework.csproj]

see: https://ci.appveyor.com/project/jogibear9988/migrator-net/build/5.0.44

There is no project.json in my directory!

**Comment by eallegretta** [comment]
I am having the exact same issue, is there a temporary solution for this?

**Comment by jogibear9988** [comment]
I've found none. I removed the old projects and used also new csproj format for the old ones (but i changed target to net4.5)

**Comment by eallegretta** [comment]
My problem is that the old project is 4.0 and cannot be changed, I'm actually trying to make the SyslogNet.Client project work as a netstandard library, but I was stopped by this random issue.

I tried modifying the MSBuild path of the old project to a Visual Studio 2013 location but with no luck.

**Comment by rainersigwald** [comment]
It looks like the problem is the existence of `obj\project.assets.json` after the `.core` project builds. I was able to work around in the original repro by moving the new core projects `obj` directory to a distinct location. Since there was a project naming convention, I conditioned the move on that.

I added a `Directory.Build.props` in the root of the repo that had this contents:

```xml
<Project>
  <PropertyGroup Condition="$(MSBuildProjectName.Contains('.core'))">
    <BaseIntermediateOutputPath>obj_core\</BaseIntermediateOutputPath>
  </PropertyGroup>
</Project>
```

@eallegretta If you have a different convention you might need to alter the condition, but I expect that this would work for you too.

I'll move this issue to the NuGet repo to see if there's a way to avoid hitting these problems, but in general it's a bit risky to share an intermediate-outputs directory among projects, so doing this split might be wise anyway.

Note that you might want to add a new pattern to `.gitignore` since this no longer obeys the standard convention.
